ro_help/hub/views.py

```diff
@@ -189,20 +189,14 @@ def get_queryset(self):
         needs = self.search(self.get_needs())
         return needs.filter(**{name: self.request.GET[name] for name in self.allow_filters if name in self.request.GET})
 
+
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         needs = self.search(self.get_needs())
         # We need a filter for the Red Cross in order to use it for the red
         # banner.
-        red_cross_filter = needs.filter(ngo__name=RED_CROSS_NAME)
-        if red_cross_filter:
-            if red_cross_filter.filter(kind=KIND.default()).exists():
-                # If we find a money need, we send it.
-                context["red_cross_context_data"] = red_cross_filter.filter(
-                    kind=KIND.default()).first()
-            else:
-                # If not, we send whatever need we get first.
-                context["red_cross_context_data"] = red_cross_filter.first()
+        context["red_cross_data"] = needs.filter(
```

Comment:
nit: What do you think of `red_cross_need`? `data` in variable's names is usually redundant since every variable holds some kind of data.